editor/components/EMSMapCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { useRef } from "react";
+import Card from "react-bootstrap/Card";
+import { MapRef } from "react-map-gl";
+import { PointMap } from "@/components/PointMap";
+import { useResizeObserver } from "@/utils/map";
+
+interface EMSMapCardProps {
+  savedLatitude: number | null;
+  savedLongitude: number | null;
+}
+
+/**
+ * Card component that renders the crash map with no editing capabilities
+ */
+export default function EMSMapCard({
+  savedLatitude,
+  savedLongitude,
+}: EMSMapCardProps) {
+  const mapRef = useRef<MapRef | null>(null);
+  /**
+   * Trigger resize() when the map container size changes - this ensures that
+   * the map repaints when the sidebar is collased/expanded.
+   */
+  const mapContainerRef = useResizeObserver<HTMLDivElement>(() => {
```

Comment:
i haven't seen this resize observer hook in action in this app yet. Very cool to see! Works just like described in your nice docs in this file. 🙏